Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```
@@ -197,15 +197,15 @@
 
 #Space Ninja Helmet
 - type: entity
-  parent: [ClothingHeadEVAHelmetBase]
+  parent: [ ClothingHeadEVAHelmetBase, ClothingHeadSuitWithLightBase ] #SS220 Ninja Resprite
```

```suggestion
  parent: [ClothingHeadSuitWithLightBase ] #SS220 Ninja Resprite
```
Лишний родитель 🐀 
Должно решить проблему с тестом, функционал останется тот же, но проверь!
```
   Failed UninitializedSaveTest [12 s]
  Error Message:
   Prototype ClothingHeadHelmetSpaceNinja modifies component on spawn: Item. Modified yaml:
heldPrefix: off
```